<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1QSVHTS" failures="2" tests="4" name="Test" time="55.008" errors="0" timestamp="2023-12-15T14:32:41 IST">
  <testcase classname="Testng.ActitimeRunnerscript" name="validLogin" time="5.656">
    <failure type="java.lang.AssertionError" message="expected [Faceboook] but found [Log in to Facebook]">
      <![CDATA[java.lang.AssertionError: expected [Faceboook] but found [Log in to Facebook]
at Testng.ActitimeRunnerscript.validLogin(ActitimeRunnerscript.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validLogin -->
  <testcase classname="Testng.ActitimeRunnerscript" name="validLogin1" time="5.663"/>
  <testcase classname="Testng.Facebook" name="validLogin" time="5.357">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Faceboook] but found [Log in to Facebook]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Faceboook] but found [Log in to Facebook]
at Testng.Facebook.validLogin(Facebook.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validLogin -->
  <testcase classname="Testng.Facebook" name="validLogin1" time="5.704"/>
</testsuite> <!-- Test -->
